app/src/main/java/com/sun/doitpat/data/model/ToDoDAO.kt

```diff
@@ -0,0 +1,21 @@
+package com.sun.doitpat.data.model
+
+import android.arch.lifecycle.LiveData
+import android.arch.persistence.room.*
+import com.sun.doitpat.util.Constants
+
+@Dao
+interface ToDoDAO {
+
+    @Query("SELECT * FROM user " + Constants.TODO_TABLE_NAME)
```

Comment:
```suggestion
    @Query("SELECT * FROM user ${Constants.TODO_TABLE_NAME}")
```
câu lệnh này thực thi gì nhỉ?

Reply:
à t chưa thay lệnh. quên mất :v